package-lock.json

```diff
@@ -33,8 +33,8 @@
       },
       "devDependencies": {
         "@playwright/test": "^1.45.3",
```

Comment:
Are changes in this file needed/expected?

Reply:
yep, package-lock is generated by running `npm install` it pins or "freezes" the dependency tree.